packages/core/src/index.js

```diff
@@ -1122,9 +1122,9 @@ export function jimpEvMethod(methodName, evName, method) {
           [methodName]: result
         });
       }
-    } catch (err) {
-      err.methodName = methodName;
-      this.emitError(methodName, err);
+    } catch (error) {
+      error.methodName = methodName;
+      this.emitErroror(methodName, error);
```

Comment:
@oliver-moran @hipstersmoothie I haven't ran into an issue with this, but I was looking over the latest release and noticed this. It looks like a find / replace error.